doc/source/tune.rst

```diff
@@ -85,7 +87,7 @@ To use rllab's VisKit (you may have to install some dependencies), run:
 
 Finally, to view the results with a `parallel coordinates visualization <https://en.wikipedia.org/wiki/Parallel_coordinates>`__, open `ParalleCoordinatesVisualization.ipynb <https://github.com/ray-project/ray/blob/master/python/ray/tune/ParallelCoordinatesVisualization.ipynb>`__ as follows and run its cells:
 
-::
+.. code-block:: bash
```

Comment:
Cathy did suggest offering a visualization here

Reply:
Let's do that as a separate task.